src/components/templates/PartyMap/components/RoomModal/RoomModal.tsx

```diff
@@ -102,7 +103,14 @@ export const RoomModalContent: React.FC<RoomModalContentProps> = ({
   });
 
   // note: this is here just to change the type on it in an easy way
-  const enterRoomWithSound: () => void = _enterRoomWithSound;
+  const enterRoomWithSound: () => void = useCallback(() => {
+    _enterRoomWithSound();
+    logEventGA({
```

Comment:
Use the full name, it's more descriptive and readable than `GA`

Reply:
By looking at the issue's description, looks like this needs to be added for the portals containing external links.
```We need to add an event for when a user clicks on a portal that goes to an external link. Let's make this a GA event to start and see if that solves the problem.``` This logic right here is going to execute always, you are not checking if the link is external or not, also this is the schedule but the issue says 'portals' which is partymap rooms.

This needs to be resolved and discussed, is it supposed to work for rooms only or schedule events as well.

There also seems to be missing 1 field called `eventValue`

Reply:
https://developers.google.com/analytics/devguides/collection/analyticsjs/field-reference#eventValue

`EventValue` can just be an Integer.

Reply:
fixed in 20d7a43cc4233cf1b6ca4b686551d874cd9034ce